app/(dashboard)/add-item/page.tsx

```diff
@@ -54,14 +54,16 @@ const AddItemPage = () => {
       ...data,
     };
 
-    const insertedItem = await insertRow('items', dataItem);
-    if (insertedItem && insertedItem.length > 0) {
-      const itemId = insertedItem[0].id;
-      console.log(itemId, 'add-item page');
+    try {
+      const addedItem = await insertRow('items', dataItem);
+      if (!addedItem || addedItem.length === 0) {
+        throw new Error('Failed to add the new item');
+      }
+      const itemId = addedItem[0].id;
       reset();
       router.push(`/add-item/success/${itemId}`);
-    } else {
-      throw new Error('Failed to add the new item');
+    } catch (error) {
+      console.error('Error adding item:', error);
```

Comment:
One final change and we're ready - can you also throw the error here please?

This way the error will be thrown from your evaluation to the `catch` block and then effectively break the page for us - which sounds counterintuitive but we want it to be flagged if any `nullish` values are coming back from the database so we can spot any future restructuring needs

Reply:
Done